connect-examples/v2/rails_payment/README.md

```diff
@@ -15,16 +15,17 @@ bundle install
 bundle exec rake db:create db:migrate # (No db in example, but keeps rails from complaining)
 ```
 
-* Add a .env file at the root with following values:
+* Update the .env file at the root with following values:
+(<b>WARNING</b>: never upload .env with your credentials/access_token)
 
 ```
 SQUARE_APPLICATION_ID=your-app-id
 SQUARE_ACCESS_TOKEN=your-access-token
 SQUARE_LOCATION_ID=your-location-id
 ```
 
-* Run Foreman: `bundle exec foreman start`
+* Run the application: `rails s`
```

Comment:
In case they have a newer Rails version installed (once this is slightly dated), it's probably a good idea to suggest `bin/rails s`, which Rails docs prefer to `bundle exec rails s`.

Reply:
Updated.